core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -515,4 +519,14 @@ class EMConfig {
     @Experimental
     @Cfg("Enable EvoMaster to generate, use, and attach complete objects to REST calls, rather than just the needed fields/values")
     var enableCompleteObjects = false
+
+    @Experimental
+    @Cfg("Whether to track individual")
```

Comment:
can you provide bit more description? eg. here I assume you are tracking the history of modifications of the individuals during the search